docs/demo/assets/styles/partials/_item.scss

```diff
@@ -55,14 +55,14 @@
 
 h2.f-item-heading {
 	@extend .dc-h2;
-	// text-transform:uppercase;
+	margin-bottom: $dc-space150;
 }
 
 h3.f-item-heading {
 	@extend .dc-h3;
 	text-transform:uppercase;
 	border-bottom:1px solid $dc-gray60;
-	// margin-bottom: $dc-space50;
+	margin-bottom: $dc-space100;
   padding-bottom: 0.4rem;
```

Comment:
Weird, I see this: 
![image](https://cloud.githubusercontent.com/assets/31034/18162278/f70b561e-7035-11e6-8cf3-20a4fb638edb.png). Maybe some kind of tabs/spaces issue with the editor, I'm checking.